backend/controller/SequenceController.ts

```diff
@@ -0,0 +1,384 @@
+import { ISequence } from '../../model/sequence/ISequence';
+import { ISlide } from '../../model/slide/ISlide';
+import { ISequenceWithSlides } from '../../model/sequence/ISequenceWithSlides';
+import { DBSequence } from '../../model/sequence/DBSequence';
+import { DBSlide } from '../../model/slide/DBSlide';
+import { LoernwerkError, LoernwerkErrorCodes } from '../loernwerkUtilities';
+import { DBUser } from '../../model/user/DBUser';
+/**
+ * Manages the sequence data in the database and handles inquiries requests regarding these
+ */
+export class SequenceController {
+    /**
+     * Creates a new sequence in the database with the given title and given user as owner.
+     * @param name the title of the sequence
+     * @param userId the userid of the owner
+     * @returns the created sequence
+     */
+    public static async createNewSequence(
+        name: string,
+        userId: number
+    ): Promise<ISequence> {
+        if ((await DBUser.findBy({ id: userId })).length !== 1) {
+            throw new LoernwerkError(
+                'user not found',
+                LoernwerkErrorCodes.NOT_FOUND
+            );
+        }
+        const seq = new DBSequence();
+        seq.creationDate = new Date();
```

Comment:
creationDate & modificationDate dont need to be set manually, they are set automatically by typeorm

Reply:
ah okay